mkv

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash -e
 go get github.com/syndtr/goleveldb/leveldb
+go get github.com/tg123/go-htpasswd
```

Comment:
Do we really need this new dependency?

Reply:
Master does a **GET** request with basic authorization to a random volume effectively performing the authorization